test/translator/read_test.cc

```diff
@@ -100,6 +107,7 @@ TEST_F(ReadTest, Read6ToNvmeShouldRead256BlocksForZeroTransferLen) {
   uint8_t network_endian_lba_1 = 0x1a;
   uint16_t network_endian_lba_2 = htons(0x2b3c);
   uint32_t cdw10 = 0x1a2b3c;
+  uint16_t transfer_len = 256;
```

Comment:
Also, what are your thoughts on just having a `kBufferIn` global variable that has a size big enough for all the tests. I think that might help make things more readable. I'm also fine just leaving it as is.